docs/hooks/resolve_references.py

```diff
@@ -74,11 +82,12 @@ def getTargetForRef(id, path):
     if not len(files):
         log.error("Unknown reference: " + id)
         return {"file": "ERROR", "title": "error"}
-    
-    file_url =   os.path.relpath(files[0], "./docs/" + os.path.dirname(path))
 
-    with open(files[0], 'r') as f:
-        content = f.read()
-        metadata = next(yaml.safe_load_all(content))
+    file_url =   os.path.relpath(files[0], "./docs/" + os.path.dirname(path))
+    data = getFileContent(files[0])
+    data["file"] = file_url
+    return data
 
-        return {"file": file_url, "title": metadata['title']}
+def on_config(config):
```

Comment:
Some icons (e.g. MASTG components, MASWE and platform) in tags are now broken

<img width="946" alt="image" src="https://github.com/user-attachments/assets/7b5b1ee8-4850-4dcd-bc6a-3ecc3b6d927c" />